cores/nRF5/Tone.cpp

```diff
@@ -370,13 +370,9 @@ bool TonePwmConfig::startPlayback(void) {
 
 bool TonePwmConfig::stopPlayback(bool releaseOwnership) {
 
-    bool notInIsr = !isInISR();
-
     if (!_HwPWM->isOwner(TonePwmConfig::toneToken)) {
-        if (notInIsr) {
-            LOG_LV2("TON", "Attempt to set noTone when not the owner of the PWM peripheral.  Ignoring call....");
-        }
-        return false;
+      LOG_LV2("TON", "Attempt to set noTone when not the owner of the PWM peripheral.  Ignoring call....");
```

Comment:
If `noTone()` called from ISR, and compiled at debug level 2, will this hang if the serial buffer is full?

Reply:
user should not call it withint ISR()

Reply:
Really?  Don't you recall the old code, which called `noTone()` from ISR to stop itself when counter was zero?

Clearly, if BSP code did this, callers could do so also.

You are changing the contract, and adding a new restriction to sketches.

**_Don't do this_**.  You will pay for this later with very difficult to track down bugs from user sketches.

Reply:
That is not that bad, the Bluetooth API is used from the start, and there should is no problem so far. User shouldn't do much processing inside any ISR(). 

PS: The previous implementation is copied from other repo, which I am too lazy to change. Now I don't see any reason to allow the API to be called in ISR.

Reply:
You are adding a new restriction ... a breaking change to the API.  I recommend against your change.

You do whatever you want.